<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HaendlerGUI</class>
 <widget class="QWidget" name="HaendlerGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/new/prefix2/cave.png</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(189, 189, 189);</string>
  </property>
  <widget class="QWidget" name="widget_haendler" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>381</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(189, 189, 189);</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lade deinen Lebens-Vorrat auf:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_schwert">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Schwert
St채rke: 7
Preis: 10 Gold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>121</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix2/haendler.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix2/donut.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix2/bonbon.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dein Kontostand:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bonbon">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Bonbon
Leben: +2
Preis: 3 Gold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_donut">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Donut
Leben: +3
Preis: 5 Gold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_futter">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Futter
St채rke: 3
Preis: 5 Gold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_leben_haendler">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Leben:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_gold_haendler">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Gold:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix2/zaubertrank.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix2/schwert.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Du hast einen H채ndler gefunden!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dein Inventar:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wehre dich gegen die Monster:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix2/futter.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_trank">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>290</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Zaubertrank
Leben: +5
Preis: 7 Gold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_platzItems">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Du hast noch Platz f체r x Items!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_item_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>235</y>
      <width>47</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 93, 93);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_item_5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>235</y>
      <width>47</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 93, 93);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_item_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>235</y>
      <width>47</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 93, 93);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_fertig">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Fertig!</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_item_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>180</y>
      <width>47</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 93, 93);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_item_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>47</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 93, 93);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_item_1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>47</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(93, 93, 93);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>365</x>
      <y>172</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(85, 0, 0);
image: url(:/new/prefix2/rahmen.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>61</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/prefix2/stein.PNG);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stein">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stein
St채rke: 5
Preis: 7 Gold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_kontostandNiedrig">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_14</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_schwert</zorder>
   <zorder>label_2</zorder>
   <zorder>label_7</zorder>
   <zorder>label_5</zorder>
   <zorder>label_9</zorder>
   <zorder>pushButton_bonbon</zorder>
   <zorder>pushButton_donut</zorder>
   <zorder>pushButton_futter</zorder>
   <zorder>label_leben_haendler</zorder>
   <zorder>label_gold_haendler</zorder>
   <zorder>label_6</zorder>
   <zorder>label_13</zorder>
   <zorder>label</zorder>
   <zorder>label_8</zorder>
   <zorder>label_3</zorder>
   <zorder>label_12</zorder>
   <zorder>pushButton_trank</zorder>
   <zorder>label_platzItems</zorder>
   <zorder>label_item_4</zorder>
   <zorder>label_item_5</zorder>
   <zorder>label_item_6</zorder>
   <zorder>pushButton_fertig</zorder>
   <zorder>label_item_3</zorder>
   <zorder>label_item_2</zorder>
   <zorder>label_item_1</zorder>
   <zorder>label_15</zorder>
   <zorder>pushButton_stein</zorder>
   <zorder>label_kontostandNiedrig</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
